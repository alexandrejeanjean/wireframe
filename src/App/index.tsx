import React from 'react'
import styled from '@emotion/styled'

import Theme from '../Theme/index'
import Main from '../components/Main'
import Header from '../components/Header'
import Footer from '../components/Footer'
import FormScene from '../scenes/FormScene'

import './App.css'

const App = () => {
  return (
    <AppStyled className='App'>
      <Theme>
        <Header />
        <Main>
          <FormScene />
        </Main>
        <Footer />
      </Theme>
    </AppStyled>
  )
}

const AppStyled = styled('div')``

export default App
