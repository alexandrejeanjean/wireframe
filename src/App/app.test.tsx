import React from 'react'
import { shallow } from 'enzyme'
import App from './index'

describe('Render App', () => {
  it('should render correctly', () => {
    const Component = shallow(<App />)
    expect(Component).toMatchSnapshot()
  })
})
