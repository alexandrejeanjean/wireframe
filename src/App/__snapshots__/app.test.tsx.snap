// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App should render correctly 1`] = `
<Styled(div)
  className="App"
>
  <Theme>
    <Main>
      <FormScene />
    </Main>
  </Theme>
</Styled(div)>
`;
